modules/persistent-storage-csi-manila-dynamic-provisioning.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-manila.adoc
+
+[id="persistent-storage-csi-manila-dynamic-provisioning_{context}"]
+= Dynamically provisioning CSI Manila volumes
+
+{product-title} installs a default StorageClass.
+
+As an application developer, you can dynamically provision RWX storage and deploy Pods with applications that safely consume the storage using YAML manifests. These YAML files are completely decoupled from Manila and from its CSI plug-in.
+
+You can use the same Pod and persistent volume claim (PVC) definitions on-premise that you use with {product-title} on AWS, GCP, Azure, and other platforms, with the exception of the storage class reference in the PVC definition.
+
+.Prerequisites
+
+* Storage must exist in the underlying infrastructure before it can be mounted as a volume in {product-title}.
+
+.UI procedure
+
+To dynamically create a CSI Manila volume using the web console:
+
+. In the {product-title} console, click *Storage* → *Persistent Volume Claims*.
+
+. In the persistent volume claims overview, click *Create Persistent Volume Claim*.
+
+. Define the required options on the resulting page.
+
+.. Select the `csi-manila-ceph` StorageClass.
+
+.. Enter a unique name for the storage claim.
+
+.. Select the access mode to specify read and write access for the PVC you are creating.
++
+[IMPORTANT]
+====
+* Use RWX if you want the persistent volume (PV) that fulfills this PVC to be mounted to multiple Pods on multiple nodes in the cluster.
+
+* Use RWO mode if you want to prevent additional Pods from being dynamically provisioned.
+====
+
+. Define the size of the storage claim.
+
+. Click *Create* to create the PersistentVolumeClaim and generate a PersistentVolume.
+
+.CLI procedure
+
+To dynamically create a CSI Manila volume using the command-line interface:
+
+. Create and save a file with the PersistentVolumeClaim object described by the following YAML:
+
+
++
+.pvc-manila.yaml
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: myclaim
+spec:
+  accessModes: <1>
+    - ReadWriteMany
+  resources:
+    requests:
+  	storage: 10Gi
+  storageClassName: csi-manila-nfs <2>
+----
++
+<1> Use RWX if you want the persistent volume (PV) that fulfills this PVC to be mounted to multiple Pods on multiple nodes in the cluster. Use RWO mode to prevent additional Pods from being dynamically provisioned.
+<2> The name of the storage class that provisions the storage back end. This was created by the cluster administrator.
++
+. Create the object you saved in the previous step by running the following command:
```

Comment:
```suggestion
. To create the object that you saved in the previous step, run the following command:
```

Reply:
@jeana-redhat, I've been using this construction, but I'm curious why _To ... run_ is preferable to _Create ... by running_; My original background is not technical writing, so it may be obvious.

Reply:
Disclaimer: I am neither an editor nor an English major, so I may have terms wrong, but will do my best ;)
This suggestion is based on a combination of favoring goal/purpose-oriented statements ("**To accomplish X**, do Y") and using the imperative to describe the action the user should take ("To accomplish X, **do Y**"). "Accomplish X by doing Y" is goal-oriented, but the imperative is on the goal rather than the actual user action. 
Personally, I don't feel there's a huge difference in meaning here, but if you look at the ISG it uses this form consistently (there's an example of this change being explicitly recommended at the end of p 225/start of p 226). There is also of course the additional benefit to translation if we all always use the same forms.

Reply:
My 2¢: 

Here's another case where minimalism, l10n, and ISG form concentric circles around doing what's best for the user, though with conflicting guidance. 

In this module, the primary goal is stated in the intro: "To dynamically create a CSI Manila volume using the command-line interface...." And I've been habituated to front-load proc sentences with verbs/actions, which is what I see most frequently in our docs ("Click Create to create the instance" instead of "To create the instance, click Create", for example).

Another issue I see with favoring goal-oriented statements is the risk of extraneous info ("Click Save" vs. "To save (?) the file, click Save"). 

Also, I've been forcing myself to use ISG and add "by running the following command(s)", so I'll add that here.

Reply:
Thanks! I of course haven't read enough of our docs to have a feel for what's really common yet :) But consistency is a good thing to add to the balance!